script.ts

```diff
@@ -18,34 +18,81 @@ To run this script, you will need a working Etherscan API key:
 If you wish to change which packs are being fetched and built, just change the `PACKS_TO_BUILD` variable near the 
 bottom of this script.
 */
-import { ABIPack, Contract } from "./types";
+import flatten from "lodash/flatten";
+import groupBy from "lodash/groupBy";
+import { ABIPack, Contract, Def, Network, TaggedAddress } from "./types";
 const fs = require("fs");
 const SDK = require("gridplus-sdk").Client;
 const superagent = require("superagent");
 const parseAbi = new SDK({ crypto: require("crypto") }).parseAbi;
 const jsonc = require("jsonc");
 const Throttle = require("superagent-throttle");
-
+const DEFAULT_NETWORK = "ethereum";
+const CONTRACT_NETWORKS = {
+  ethereum: {
+    baseUrl: "https://api.etherscan.io",
+    apiKey: process.env.ETHERSCAN_KEY,
```

Comment:
Just curious... where does this get defined in CI?

Reply:
I meant to mention that. We add them in the settings of the repo itself. I put in the Etherscan key that I generated, but we might want to update that to an official one, and we'll need to add keys for the other services.